config.js

```diff
@@ -312,23 +304,27 @@ if (argv.id) {
     console.log("Warning: --id alias is no longer supported. Use --username instead.");
 }
 
-if (argv.minrankedhandicap) {
-    console.log("Warning: --minrankedhandicap argument is no longer supported. Use --minhandicapranked instead.");
+if (argv.minrankedhandicap || argv.minunrankedhandicap || argv.maxrankedhandicap || argv.maxunrankedhandicap) {
+    console.log("Warning: --min/max*+/-ranked/unranked*handicap argument is no longer supported.\nUse --min/max*handicap*+/-ranked/unranked instead.");
```

Comment:
I liked it when it gave one error per modified argument. 

Also, i see a pattern for a function here.

```
function checkParam(b, old, new) {
  if (b) {
    console.log(`Warning: --${old} argument is no longer supported.\nUse --${new} instead.`);
  }
}
```

You can even declare a list here and run the function many times:

```
[[argv.minrankedhandicap, "minrankedhandicap", "minhandicapranked"],
[argv.maxrankedhandicap, "maxrankedhandicap", "maxhandicapranked"]
[argv.minunrankedhandicap, "minunrankedhandicap", "minhandicapunranked"],
[argv.maxunrankedhandicap, "maxunrankedhandicap", "maxhandicapunranked"]]
  .forEach(ar=>checkParam(...ar))
```

Reply:
interesting too, but first priority is to fix not working things or add new features :+1: 

but always thanks for teaching me and everyone things !!